tanka-util/main.libsonnet

```diff
@@ -0,0 +1,33 @@
+local d = import 'github.com/sh0rez/docsonnet/doc-util/main.libsonnet';
+{
+  local this = self,
+
+  '#':: d.pkg(
+    name='tanka_util',
+    url='github.com/grafana/jsonnet-libs/tanka-util/main.libsonnet',
+    help=(importstr 'README.md.tmpl') % (importstr '_example.jsonnet'),
+  ),
+
+  '#util':: d.obj(
+    |||
+      `util` provides common utils to modify Kubernetes objects.
+    |||
+  ),
+  util: (import 'util.libsonnet'),
```

Comment:
Can we name this `k8s` instead? `util` is a term that carris few useful information